versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CachingPersistImpl.java

```diff
@@ -132,49 +132,20 @@ public Obj[] fetchObjs(@Nonnull @jakarta.annotation.Nonnull ObjId[] ids)
   }
 
   @Override
-  public boolean storeObj(
+  public void storeObj(
       @jakarta.annotation.Nonnull @Nonnull Obj obj, boolean ignoreSoftSizeRestrictions)
       throws ObjTooLargeException {
-    if (persist.storeObj(obj, ignoreSoftSizeRestrictions)) {
-      cache.put(obj);
-      return true;
-    }
-    return false;
-  }
-
-  @Override
-  @Nonnull
-  @jakarta.annotation.Nonnull
-  public boolean[] storeObjs(@jakarta.annotation.Nonnull @Nonnull Obj[] objs)
-      throws ObjTooLargeException {
-    boolean[] stored = persist.storeObjs(objs);
-    for (int i = 0; i < stored.length; i++) {
-      if (stored[i]) {
-        cache.put(objs[i]);
-      }
-    }
-    return stored;
+    persist.storeObj(obj, ignoreSoftSizeRestrictions);
+    cache.put(obj);
   }
 
   @Override
-  public void upsertObj(@jakarta.annotation.Nonnull @Nonnull Obj obj) throws ObjTooLargeException {
-    try {
-      persist.upsertObj(obj);
-    } finally {
-      cache.remove(obj.id());
```

Comment:
I don't understand why the upsert methods were removing objects from the cache. I think it's an oversight.